contents/docs/feature-flags/local-evaluation.mdx

```diff
@@ -0,0 +1,174 @@
+---
+title: Server-side local evaluation
+sidebar: Docs
+showTitle: true
+availability:
+    free: full
+    selfServe: full
+    enterprise: full
+---
+
+> **Note:** Local evaluation is only available in the [Node](/docs/libraries/node), [Ruby](https://posthog.com/docs/libraries/ruby), [Go](https://posthog.com/docs/libraries/go), and [Python](https://posthog.com/docs/libraries/python) SDKs.
+
+> **Note:** Do not use local evaluation in an edge / lambda environment, as this can slow down performance, and also raise your bill drastically. It's best to use regular flag evaluation instead.
+
+Evaluating feature flags requires making a request to PostHog for each flag. However, you can improve performance by evaluating flags locally.  Instead of making a request for each flag, PostHog will periodically request and store feature flag definitions locally, enabling you to evaluate flags without making additional requests.
+
+It is best practice to use local evaluation flags when possible, since this enables you to resolve flags faster and with fewer API calls.
```

Comment:
We could explain this better, specifically the benefits of local evaluation and when / why you should implement it vs the "standard" implementation. This feels vague and hand wavy.

Should we also touch on how this impacts how you're charged? Afaik, we technically charge more for local eval, but it should cost users less because it's more efficient.

Reply:
> We could explain this better, specifically the benefits of local evaluation and when / why you should implement it vs the "standard" implementation. This feels vague and hand wavy.

I think this is sufficient. It's common enough knowledge that making a request means you have to wait for it to return, so I dont think we need to go into more detail here.

> Should we also touch on how this impacts how you're charged? Afaik, we technically charge more for local eval, but it should cost users less because it's more efficient.

Touched on this below